lib/rubocop/cop/betterment/non_standard_actions.rb

```diff
@@ -60,7 +60,7 @@ def check_raw_route(node)
         end
 
         def allowed_actions
-          @allowed_actions ||= cop_config["AllowedActions"]
+          @allowed_actions ||= cop_config['StandardActions'] + cop_config['AdditionalAllowedActions']
```

Comment:
Worth double checking that `cop_config` is effectively a `merge` between the local YML and the upstream default YML -- I think I've seen in the past that, for example, defining something in `.rubocop.yml` will completely override any generated defaults in `.rubocop_todo.yml`, so in this case the consumer may still have to redefine all `StandardActions` if they want to set any `AdditionalAllowedActions`.

This might require you to do something like `standard_actions = cop_config['StandardActions'] || DEFAULT_STANDARD_ACTIONS` rather than relying on the default YML for that list.

Reply:
Yep! Tested the example in the readme and it works